infra/prover/index.ts

```diff
@@ -3,15 +3,15 @@ import * as aws from '@pulumi/aws';
 import * as awsx from '@pulumi/awsx';
 import * as docker_build from '@pulumi/docker-build';
 import * as pulumi from '@pulumi/pulumi';
-import { getEnvVar } from "./util/env";
+import { getEnvVar, ChainId, getServiceNameV1 } from "../util";
 
 export = () => {
   // Read config
   const config = new pulumi.Config();
 
-  const isDev = pulumi.getStack() === "dev";
-  const prefix = isDev ? `${getEnvVar("DEV_NAME")}-` : "";
-  const serviceName = `${prefix}bonsai-prover`;
+  const stackName = pulumi.getStack();
+  const isDev = stackName === "dev";
```

Comment:
it doesn't look you're you're using `isDev`?

Reply:
Using it on L25+

Reply:
oh, should have looked on the left 🤦🏽